.circleci/config.yml

```diff
@@ -41,9 +41,10 @@ executors:
     docker:
       - image: okteto/golang-ci:2.7.0
     environment:
-      OKTETO_CONTEXT: https://staging.okteto.dev
-      OKTETO_USER: cindylopez
-      OKTETO_APPS_SUBDOMAIN: staging.okteto.dev
+      OKTETO_CONTEXT: https://okteto.staging.dev.okteto.net
```

Comment:
When migration is finished, this will be https://staging.okteto.dev again. But we need this change while both environments are running